src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,11 +765,41 @@ export default {
 }
 
 .conversations-search {
-	flex-grow: 1;
+	transition: all 0.3s ease;
+	width: 65%;
+	z-index: 2;
+	position : absolute;
 
 	:deep(.input-field__input) {
 		border-radius: var(--border-radius-pill);
 	}
+	&--expanded {
+		width: 90%;
+	}
+
+	&--filter{
```

Comment:
So that means, we cannot use filters and search together? What if I want to find all unread conversation with 'Talk' in theirs name?
I think we need a two-layer filtration here, but maybe I missed something in the discussion